packages/atlas/src/components/_comments/ReactionChip/ReactionChip.tsx

```diff
@@ -1,46 +1,94 @@
-import React from 'react'
+import React, { useEffect, useRef } from 'react'
 
 import { Text } from '@/components/Text'
 import { Loader } from '@/components/_loaders/Loader'
+import { PopoverImperativeHandle } from '@/components/_overlays/Popover'
+import { ReactionsOnboardingPopover } from '@/components/_video/ReactionsOnboardingPopover'
+import { pluralizeNoun } from '@/utils/misc'
 import { formatNumberShort } from '@/utils/number'
 
 import { EmojiContainer, ReactionChipButton } from './ReactionChip.styles'
 
-export type ReactionType = 'amusment' | 'love' | 'laugh' | 'shock' | 'anger'
+export type ReactionId = 1 | 2 | 3 | 4 | 5
+
+export type ReactionType = 'like' | 'love' | 'laugh' | 'shock' | 'anger'
```

Comment:
`ReactionId` is what we're getting from QN. This is not very useful, to be honest - I think the query node should return something like an enum, maybe? 
Anyway, I changed my approach a bit and I created an object which has ReactionId as a property and object as a value, which contains the name and the corresponding emoji. I moved this to @/config/reactions.ts
A reaction name is used to add a more accessible name to `ReactionChip` button - you can check the `title` prop. Let me know what you think

Reply:
We specifically didn't want to make it an enum so that apps are free to pick whatever reactions they want to use. Somebody may fork Atlas and use 🤮 as reaction instead and they should be free to do so :P It's our job as app developers to keep the list of reactions we want to use and assign some ids to them. Will take a look at the code again